Webspage 3.0/index.htm

```diff
@@ -1,15 +1,21 @@
 <!DOCTYPE html>
 <html>
 <head>
-<link rel="stylesheet" type="text/css" href="mystyles.css">
+<link rel="stylesheet" type="text/css" href="styles/main.css">
 </head>
 
 <body>
 
-<img src="img/gauge_menu.png" style="float:right; width: 275px; height: 500px;">
-<h1>Mike Elrod</h1>
-<h2>Portfolio</h2>
 
+	<h1>MIKE ELROD</h1>
+	<h2>PORTFOLIO</h2>
+
+	<div>
+		web
+		video
+		graphics
+		photo
+    </div>

```

Comment:
Keep your indentations consistent. When you have a lot of code on the page, you need to be able to scan it easily.